ykrt/src/compile/jitc_llvm.rs

```diff
@@ -36,7 +36,13 @@ impl Compiler for JITCLLVM {
         sti: Option<SideTraceInfo>,
         hl: Arc<Mutex<HotLocation>>,
     ) -> Result<CompiledTrace, CompilationError> {
-        let irtrace = aottrace_iter.0.collect::<Vec<_>>();
+        let mut irtrace = Vec::new();
+        for ta in aottrace_iter.0 {
+            match ta {
+                Ok(x) => irtrace.push(x),
+                Err(_) => todo!(),
```

Comment:
Why not populate the `todo!()`s? Should be a simple `.map_err()`?

Reply:
I have no way of testing these clauses, and I am not keen on adding more untested code to yk. I would like to address hwt's lack of testing, which will hopefully shake out `todo`s and ensure we actually do the right thing.

Reply:
To test this you can provide virtual addresses that are not in the block map. Probably no small change though, so perhaps another PR at some point?

Reply:
Yes, we definitely need to think how we can test hwt more thoroughly (e.g. unit tests with PT data). But that's out of scope for now IMHO.